app/templates/Streunerkatzen/Includes/Mail/CatEntryReminderMail.ss

```diff
@@ -0,0 +1,17 @@
+<h1>Hi!</h1>
+
+<p>
+    Du erhältst diese E-Mail, weil du eine oder mehrere Katzen auf <a href="$BaseHref">streunerkatzen.org</a> eingetragen hast. Du kannst weiter unten deine Benachrichtigungseinstellungen für jede Katze einzeln festlegen.
```

Comment:
should we make this configurable? if yes, how exactly? where/how do we add the list of cats?